packages/react-select/src/Select.js

```diff
@@ -806,7 +806,7 @@ export default class Select extends Component<Props, State> {
     const custom = this.props.styles[key];
     return custom ? custom(base, props) : base;
   };
-  getElementId = (element: 'group' | 'input' | 'listbox' | 'option') => {
+  getElementId = (element: 'group' | 'input' | 'listbox' | 'combobox' | 'option') => {
```

Comment:
I'm not a fan of this approach and I understand you are adapting to existing code.

This should likely just be `menu` so we don't have to know the role of the element to know its id.